Common/src/CConfig.cpp

```diff
@@ -1058,7 +1058,6 @@ void CConfig::SetPointersNull(void) {
   Kind_TimeNumScheme = EULER_IMPLICIT;
 
   Gas_Composition = nullptr;
-
 }
```

Comment:
I know these sneak in: unrelated empty line removals or  additions, but additional style points are given if they aren't. small cleanups in related sections are fine though

Reply:
Thanks!!!